.github/workflows/built-release.yml

```diff
@@ -0,0 +1,18 @@
+name: Built Branch and Release
+
+on:
+  push:
+    branches:
+      - develop
+      - main
+      - production
```

Comment:
do we want to build a release on all these braches? `develop` and `production` both exist.